tidy3d/web/httputils.py

```diff
@@ -1,15 +1,59 @@
 """ handles communication with server """
-# import os
+import os
 import time
+from os.path import expanduser
 from typing import Dict
 from enum import Enum
 
 import jwt
 import requests
+import toml
 
 from .auth import get_credentials, MAX_ATTEMPTS
 from .config import DEFAULT_CONFIG as Config
 from ..log import WebError
+from ..version import __version__
+
+SIMCLOUD_APIKEY = "SIMCLOUD_APIKEY"
+
+
+def api_key():
+    """
+    Get the api key for the current environment.
+    :return:
+    """
+    if os.environ.get(SIMCLOUD_APIKEY):
+        return os.environ.get(SIMCLOUD_APIKEY)
+    if os.path.exists(f"{expanduser('~')}/.tidy3d/config"):
```

Comment:
need to `from .cli.app import CONFIG_FILE` and use it here and int he line below?

Reply:
I think it would be better to have creator manage the constant, in this case, we could leave it in ``app.py``

Reply:
agree